publish.sh

```diff
@@ -82,6 +83,19 @@ if [ "$PROJECT" == "orm" ]; then
 		-Pgradle.publish.key=$PLUGIN_PORTAL_USERNAME -Pgradle.publish.secret=$PLUGIN_PORTAL_PASSWORD \
 		-PhibernatePublishUsername=$OSSRH_USER -PhibernatePublishPassword=$OSSRH_PASSWORD \
 		-DsigningPassword=$RELEASE_GPG_PASSPHRASE -DsigningKeyFile=$RELEASE_GPG_PRIVATE_KEY_PATH --stacktrace
+
+elif [ "$PROJECT" == "reactive" ]; then
+	git config user.email ci@hibernate.org
+	git config user.name Hibernate-CI
+
+	# Update the project development version and create the tag for the version to be released
+	exec_or_dry_run ./gradlew ciRelease -PreleaseVersion=$RELEASE_VERSION -PdevelopmentVersion=$DEVELOPMENT_VERSION -PgitRemote=origin -PgitBranch=$BRANCH
+	# Create the artifacts and the documentation
+	exec_or_dry_run ./gradlew assemble
```

Comment:
Ideally this would be done earlier, so that a failure would not require you to revert anything (e.g. deleting the tag pushed just above).